src/app/services/actions/permissions-action-creator.ts

```diff
@@ -13,6 +12,12 @@ import {
   FETCH_SCOPES_PENDING,
   FETCH_SCOPES_SUCCESS,
 } from '../redux-constants';
+import {
+  WORK,
```

Comment:
These can be captured as Enum 

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
thanks for the suggestion! using enum makes alot more sense 😊